data-es2016plus.js

```diff
@@ -824,7 +824,7 @@ exports.tests = [
           firefox53: firefox.sharedmem,
           firefox55: true,
           chrome48: chrome.sharedmem,
-          chrome61: true,
```

Comment:
shared memory was turned on by default in v8 6.0 just recently (when it became beta)